db/migrations/20230307112509_order_items.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE order_items(
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  item_id INT NOT NULL,
+  item_quantity INT NOT NULL,
+  order_id INT(100) NOT NULL,
```

Comment:
```suggestion
  order_id INT NOT NULL,
```
fkey 아닌가요?? 또한 mysql 에서 INT field는 지정하는 자릿수에 대한 상관없이 자신이 수용할 수 있는 크기 만큼 받는걸로 알고 있습니다!